tests/FSharp.Compiler.ComponentTests/FSharp.Compiler.ComponentTests.fsproj

```diff
@@ -19,9 +19,10 @@
   </ItemGroup>
```

Comment:
These tests can run just on `netcoreapp3.1`.  We have enough coverage on `net472` through visual studio tests and other `test/fsharp` tests

Reply:
@dsyme This is the fsharpqa suite.  For sure they run on desktop, if we can't make them run net472 then we are doing something wrong.